recipes/suitesparse/SuiteSparse_config_Mac.mk

```diff
@@ -0,0 +1,452 @@
+
+# This file contains all configuration settings for all packages authored or
+# co-authored by Tim Davis:
+#
+# Package Version       Description
+# ------- -------       -----------
+# AMD     1.2 or later  approximate minimum degree ordering
+# COLAMD  2.4 or later  column approximate minimum degree ordering
+# CCOLAMD 1.0 or later  constrained column approximate minimum degree ordering
+# CAMD    any           constrained approximate minimum degree ordering
+# UMFPACK 4.5 or later  sparse LU factorization, with the BLAS
+# CHOLMOD any           sparse Cholesky factorization, update/downdate
+# KLU     0.8 or later  sparse LU factorization, BLAS-free
+# BTF     0.8 or later  permutation to block triangular form
+# LDL     1.2 or later  concise sparse LDL'
+# CXSparse any          extended version of CSparse (int/long, real/complex)
+# SuiteSparseQR any     sparse QR factorization
+# RBio    2.0 or later  read/write sparse matrices in Rutherford-Boeing format
+#
+# By design, this file is NOT included in the CSparse makefile.
+# That package is fully stand-alone.  CSparse is primarily for teaching;
+# production code should use CXSparse.
+#
+# The SuiteSparse_config directory and the above packages should all appear in
+# a single directory, in order for the Makefile's within each package to find
+# this file.
+#
+# To enable an option of the form "# OPTION = ...", edit this file and
+# delete the "#" in the first column of the option you wish to use.
+#
+# The use of METIS 4.0.1 is optional.  To exclude METIS, you must compile with
+# CHOLMOD_CONFIG set to -DNPARTITION.  See below for details.  However, if you
+# do not have a metis-4.0 directory inside the SuiteSparse directory, the
+# */Makefile's that optionally rely on METIS will automatically detect this
+# and compile without METIS.
+
+#------------------------------------------------------------------------------
+# Generic configuration
+#------------------------------------------------------------------------------
+
+# Using standard definitions from the make environment, typically:
+#
+#   CC              cc      C compiler
+#   CXX             g++     C++ compiler
+#   CFLAGS          [ ]     flags for C and C++ compiler
+#   CPPFLAGS        [ ]     flags for C and C++ compiler
+#   TARGET_ARCH     [ ]     target architecture
+#   FFLAGS          [ ]     flags for Fortran compiler
+#   RM              rm -f   delete a file
+#   AR              ar      create a static *.a library archive
+#   ARFLAGS         rv      flags for ar
+#   MAKE            make    make itself (sometimes called gmake)
+#
+# You can redefine them here, but by default they are used from the
+# default make environment.
+
+# To use OpenMP add -openmp to the CFLAGS
+# If OpenMP is used, it is recommended to define CHOLMOD_OMP_NUM_THREADS
+# as the number of cores per socket on the machine being used to maximize
+# memory performance
+  CFLAGS = 
```

Comment:
setting the following was needed for me to find the TBB headers at the INCLUDE_PATH defined in build.sh
CFLAGS = -I$(INCLUDE_PATH)